Formula/tt.rb

```diff
@@ -24,6 +24,10 @@ class Tt < Formula
   uses_from_macos "unzip"
   uses_from_macos "zip"
 
+  on_macos do
+    depends_on "bash-completion"
+  end
+
```

Comment:
is this hard dependency?

Reply:
HI. Yes, it is hard dependency. Module 'tt completion bash' won't work without it.

Reply:
Sounds good, thanks!